health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/PopUpConfigEditor.js

```
@@ -198,23 +198,18 @@ const PopupLabelField = ({ label, path, value, selectedField }) => {
       {/* Label on top */}
-      <label
-        style={{
-          fontSize: "14px",
-          fontWeight: "500",
-          color: "#0B0C0C",
-        }}
-      >
-        {t(label)}
-      </label>
+      <div style={{ display: "flex", marginBottom: "8px" }}>
+        <label style={{ fontWeight: "500", fontSize: "14px" }}>{t(label)}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)